src/rules/format/repeatcount-format-rule.js

```diff
@@ -0,0 +1,53 @@
+const Rule = require('../rule');
+const ValidationErrorType = require('../../errors/validation-error-type');
+const ValidationErrorCategory = require('../../errors/validation-error-category');
+const ValidationErrorSeverity = require('../../errors/validation-error-severity');
+
+module.exports = class RepeatCountIsPositiveInteger extends Rule {
```

Comment:
Is this rule now a total duplicate of https://github.com/openactive/data-model-validator/pull/391 ? Should it be removed?

The integer test should already be handled by the data type.

Hence the error returned by [this data](https://validator.openactive.io/?version=2.x&validationMode=RPDEFeed#/json/ewogICJuZXh0IjogImh0dHBzOi8vd3d3LmV4YW1wbGUuY29tL2FwaS9ycGRlL3Nlc3Npb24tc2VyaWVzP2FmdGVyVGltZXN0YW1wPTE1MjE1NjU3MTkmYWZ0ZXJJZD0xNDAyQ0JQMjAxNTAyMTciLAogICJpdGVtcyI6IFsKICAgIHsKICAgICAgInN0YXRlIjogInVwZGF0ZWQiLAogICAgICAia2luZCI6ICJTZXNzaW9uU2VyaWVzIiwKICAgICAgImlkIjogIjE0MDJDQlAyMDE1MDIxNyIsCiAgICAgICJtb2RpZmllZCI6IDE1MjE1NjU3MTksCiAgICAgICJkYXRhIjogewogICAgICAgICJAY29udGV4dCI6ICJodHRwczovL29wZW5hY3RpdmUuaW8vIiwKICAgICAgICAiQGlkIjogImh0dHBzOi8vb3BlbnNlc3Npb25zLmlvL2FwaS9zZXNzaW9uLXNlcmllcy8xNDAyQ0JQMjAxNTAyMTciLAogICAgICAgICJpZGVudGlmaWVyIjogIjE0MDJDQlAyMDE1MDIxNyIsCiAgICAgICAgIkB0eXBlIjogIlNlc3Npb25TZXJpZXMiLAogICAgICAgICJvcmdhbml6ZXIiOiB7CiAgICAgICAgICAiQHR5cGUiOiAiT3JnYW5pemF0aW9uIiwKICAgICAgICAgICJAaWQiOiAiaHR0cHM6Ly9pZC5ib29raW5nc3lzdGVtLmV4YW1wbGUuY29tL29yZ2FuaXplcnMvMSIsCiAgICAgICAgICAibmFtZSI6ICJFdmVyeW9uZSBBY3RpdmUiLAogICAgICAgICAgInVybCI6ICJodHRwczovL3d3dy5ldmVyeW9uZWFjdGl2ZS5jb20vIiwKICAgICAgICAgICJsb2dvIjogewogICAgICAgICAgICAiQHR5cGUiOiAiSW1hZ2VPYmplY3QiLAogICAgICAgICAgICAidXJsIjogImh0dHBzOi8vczMtZXUtd2VzdC0yLmFtYXpvbmF3cy5jb20vcHJvZC1ldmVyeW9uZWFjdGl2ZS13cC93cC1jb250ZW50L3VwbG9hZHMvMjAxNy8xMC8yNjA5MjIzNi9FYS1JbWFnZTIuanBnIgogICAgICAgICAgfSwKICAgICAgICAgICJlbWFpbCI6ICJjdXN0b21lcnNlcnZpY2VzQGdsbC5vcmciLAogICAgICAgICAgInRlbGVwaG9uZSI6ICIwMTQ1NSA4OTA1MDgiLAogICAgICAgICAgInNhbWVBcyI6IFsKICAgICAgICAgICAgImh0dHBzOi8vd3d3LmZhY2Vib29rLmNvbS9ldmVyeW9uZWFjdGl2ZS8iLAogICAgICAgICAgICAiaHR0cHM6Ly90d2l0dGVyLmNvbS9ldmVyeW9uZWFjdGl2ZSIKICAgICAgICAgIF0KICAgICAgICB9LAogICAgICAgICJhY3Rpdml0eSI6IFsKICAgICAgICAgIHsKICAgICAgICAgICAgIkB0eXBlIjogIkNvbmNlcHQiLAogICAgICAgICAgICAiQGlkIjogImh0dHBzOi8vb3BlbmFjdGl2ZS5pby9hY3Rpdml0eS1saXN0IzVlNzhiY2JlLTM2ZGItNDI1YS05MDY0LWJmOTZkMDljYzM1MSIsCiAgICAgICAgICAgICJwcmVmTGFiZWwiOiAiQm9keXB1bXDihKIiLAogICAgICAgICAgICAiaW5TY2hlbWUiOiAiaHR0cHM6Ly9vcGVuYWN0aXZlLmlvL2FjdGl2aXR5LWxpc3QiCiAgICAgICAgICB9CiAgICAgICAgXSwKICAgICAgICAiYWNjZXNzaWJpbGl0eVN1cHBvcnQiOiBbCiAgICAgICAgICB7CiAgICAgICAgICAgICJAdHlwZSI6ICJDb25jZXB0IiwKICAgICAgICAgICAgIkBpZCI6ICJodHRwczovL29wZW5hY3RpdmUuaW8vYWNjZXNzaWJpbGl0eS1zdXBwb3J0IzEzOTNmMmRjLTNmY2MtNGJlOS1hOTlmLWYxZTUxZjVhZDI3NyIsCiAgICAgICAgICAgICJwcmVmTGFiZWwiOiAiVmlzdWFsIGltcGFpcm1lbnQiLAogICAgICAgICAgICAiaW5TY2hlbWUiOiAiaHR0cHM6Ly9vcGVuYWN0aXZlLmlvL2FjY2Vzc2liaWxpdHktc3VwcG9ydCIKICAgICAgICAgIH0KICAgICAgICBdLAogICAgICAgICJjYXRlZ29yeSI6IFsKICAgICAgICAgICJHcm91cCBFeGVyY2lzZSBDbGFzc2VzIiwKICAgICAgICAgICJUb25pbmcgJiBTdHJlbmd0aCIsCiAgICAgICAgICAiR3JvdXAgRXhlcmNpc2UgLSBWaXJ0dWFsIgogICAgICAgIF0sCiAgICAgICAgIm5hbWUiOiAiVmlydHVhbCBCT0RZUFVNUCIsCiAgICAgICAgImRlc2NyaXB0aW9uIjogIlRoaXMgaXMgdGhlIHZpcnR1YWwgdmVyc2lvbiBvZiB0aGUgb3JpZ2luYWwgYmFyYmVsbCBjbGFzcywgd2hpY2ggd2lsbCBoZWxwIHlvdSBnZXQgbGVhbiwgdG9uZWQgYW5kIGZpdCAtIGZhc3QuIExlcyBNaWxsc-KEoiBWaXJ0dWFsIGNsYXNzZXMgYXJlIGRlc2lnbmVkIGZvciBwZW9wbGUgd2hvIGNhbm5vdCBnZXQgYWNjZXNzIHRvIG91ciBsaXZlIGNsYXNzZXMgb3Igd2hvIHdhbnQgdG8gZ2V0IGEg4oCYdGFzdGXigJkgb2YgYSBMZXMgTWlsbHPihKIgY2xhc3MgYmVmb3JlIHRha2luZyBhIGxpdmUgY2xhc3Mgd2l0aCBhbiBpbnN0cnVjdG9yLiBUaGUgY2xhc3NlcyBhcmUgcGxheWVkIG9uIGEgYmlnIHZpZGVvIHNjcmVlbiwgd2l0aCBkaW1tZWQgbGlnaHRpbmcgYW5kIHB1bXBpbmcgc3Vycm91bmQgc291bmQsIGFuZCBhcmUgbGVkIG9uc2NyZWVuIGJ5IHRoZSBwZW9wbGUgd2hvIGFjdHVhbGx5IGNob3Jlb2dyYXBoIHRoZSBjbGFzc2VzLiIsCiAgICAgICAgImdlbmRlclJlc3RyaWN0aW9uIjogImh0dHBzOi8vb3BlbmFjdGl2ZS5pby9Ob1Jlc3RyaWN0aW9uIiwKICAgICAgICAiYWdlUmFuZ2UiOiB7CiAgICAgICAgICAiQHR5cGUiOiAiUXVhbnRpdGF0aXZlVmFsdWUiLAogICAgICAgICAgIm1pblZhbHVlIjogMTYKICAgICAgICB9LAogICAgICAgICJsZXZlbCI6IFsKICAgICAgICAgICJCZWdpbm5lciIKICAgICAgICBdLAogICAgICAgICJpbWFnZSI6IFsKICAgICAgICAgIHsKICAgICAgICAgICAgIkB0eXBlIjogIkltYWdlT2JqZWN0IiwKICAgICAgICAgICAgInVybCI6ICJodHRwczovL3MzLWV1LXdlc3QtMi5hbWF6b25hd3MuY29tL3Byb2QtZXZlcnlvbmVhY3RpdmUtd3Avd3AtY29udGVudC91cGxvYWRzLzIwMTcvMTAvMjYwOTIyMzYvRWEtSW1hZ2UyLmpwZyIKICAgICAgICAgIH0KICAgICAgICBdLAogICAgICAgICJ1cmwiOiAiaHR0cHM6Ly9wcm9maWxlLmV2ZXJ5b25lYWN0aXZlLmNvbS9ib29raW5nP1NpdGU9MDE0MCZBY3Rpdml0aWVzPTE0MDJDQlAyMDE1MDIxNyZDdWx0dXJlPWVuLUdCIiwKICAgICAgICAibGVhZGVyIjogWwogICAgICAgICAgewogICAgICAgICAgICAiQHR5cGUiOiAiUGVyc29uIiwKICAgICAgICAgICAgImZhbWlseU5hbWUiOiAiU21pdGgiLAogICAgICAgICAgICAiZ2l2ZW5OYW1lIjogIk5pY29sZSIKICAgICAgICAgIH0KICAgICAgICBdLAogICAgICAgICJsb2NhdGlvbiI6IHsKICAgICAgICAgICJAdHlwZSI6ICJQbGFjZSIsCiAgICAgICAgICAidXJsIjogImh0dHBzOi8vd3d3LmV2ZXJ5b25lYWN0aXZlLmNvbS9jZW50cmVzL01pZGRsZXNicm91Z2gtU3BvcnRzLVZpbGxhZ2UiLAogICAgICAgICAgIm5hbWUiOiAiTWlkZGxlc2Jyb3VnaCBTcG9ydHMgVmlsbGFnZSIsCiAgICAgICAgICAiZGVzY3JpcHRpb24iOiAiWW91IGNhbiBlbmpveSBhIGh1Z2UgcmFuZ2Ugb2YgZGlmZmVyZW50IHNwb3J0cyBhbmQgYWN0aXZpdGllcyBhdCBNaWRkbGVzYnJvdWdoIFNwb3J0cyBWaWxsYWdlLiBUaGVyZeKAmXMgYSBzdGF0ZS1vZi10aGUtYXJ0IGd5bSwgYSBmdWxsIHByb2dyYW1tZSBvZiBncm91cCBleGVyY2lzZSBjbGFzc2VzLCBpbmRvb3IgYmFkbWludG9uIHNlc3Npb25zIGFuZCBhIHNvZnQgcGxheSBhcmVhIGZvciBraWRzLiBPdXRzaWRlLCB0aGUgVmlsbGFnZSBib2FzdHMgYSBmdWxsIHJhbmdlIG9mIGF0aGxldGljcyBmYWNpbGl0aWVzLCBpbmNsdWRpbmcgYW4gZWlnaHQtbGFuZSA0MDBtIHRyYWNrLCBhIDEwLWxhbmUgMTAwbSB0cmFjaywgYXMgd2VsbCBhcyBmb3VyIGxvbmcganVtcCBwaXRzLiBUaGVyZeKAmXMgYWxzbyBoaWdoIGp1bXAsIGhhbW1lciB0aHJvd2luZywgamF2ZWxpbiBhbmQgcG9sZSB2YXVsdGluZyBmYWNpbGl0aWVzLiBUaGUgbGF0ZXN0IGFkZGl0aW9uIHRvIHRoZSBmYWNpbGl0eSBpcyBhIMKjMS42bSwgMjUwbSB2ZWxvZHJvbWUsIGFsb25nc2lkZSBmaXZlIGFsbC13ZWF0aGVyIG91dGRvb3IgcGl0Y2hlcyBhbmQgYSBza2F0ZSBwYXJrIGZvciBCTVggYmlrZXMsIHNjb290ZXJzIGFuZCBpbmxpbmUgc2thdGVycy4iLAogICAgICAgICAgImlkZW50aWZpZXIiOiAiMDE0MCIsCiAgICAgICAgICAiYWRkcmVzcyI6IHsKICAgICAgICAgICAgIkB0eXBlIjogIlBvc3RhbEFkZHJlc3MiLAogICAgICAgICAgICAic3RyZWV0QWRkcmVzcyI6ICJBbGFuIFBlYWNvY2sgV2F5IiwKICAgICAgICAgICAgImFkZHJlc3NMb2NhbGl0eSI6ICJWaWxsYWdlIEVhc3QiLAogICAgICAgICAgICAiYWRkcmVzc1JlZ2lvbiI6ICJNaWRkbGVzYnJvdWdoIiwKICAgICAgICAgICAgInBvc3RhbENvZGUiOiAiVFM0IDNBRSIsCiAgICAgICAgICAgICJhZGRyZXNzQ291bnRyeSI6ICJHQiIKICAgICAgICAgIH0sCiAgICAgICAgICAidGVsZXBob25lIjogIjAxNjQyIDcyODU1NSIsCiAgICAgICAgICAiZ2VvIjogewogICAgICAgICAgICAiQHR5cGUiOiAiR2VvQ29vcmRpbmF0ZXMiLAogICAgICAgICAgICAibGF0aXR1ZGUiOiA1NC41NDM5NjQsCiAgICAgICAgICAgICJsb25naXR1ZGUiOiAtMS4yMDk3ODUwMDAwMDAwMQogICAgICAgICAgfSwKICAgICAgICAgICJpbWFnZSI6IFsKICAgICAgICAgICAgewogICAgICAgICAgICAgICJAdHlwZSI6ICJJbWFnZU9iamVjdCIsCiAgICAgICAgICAgICAgInVybCI6ICJodHRwczovL3MzLWV1LXdlc3QtMi5hbWF6b25hd3MuY29tL3Byb2QtZXZlcnlvbmVhY3RpdmUtd3Avd3AtY29udGVudC91cGxvYWRzLzIwMTgvMDQvMTMxMTIyMTUvQWN0b25fSGVhZGVyQmFubmVyLnBuZyIKICAgICAgICAgICAgfQogICAgICAgICAgXSwKICAgICAgICAgICJhbWVuaXR5RmVhdHVyZSI6IFsKICAgICAgICAgICAgewogICAgICAgICAgICAgICJuYW1lIjogIkNoYW5naW5nIEZhY2lsaXRpZXMiLAogICAgICAgICAgICAgICJ2YWx1ZSI6IHRydWUsCiAgICAgICAgICAgICAgIkB0eXBlIjogIkNoYW5naW5nRmFjaWxpdGllcyIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgewogICAgICAgICAgICAgICJuYW1lIjogIlNob3dlcnMiLAogICAgICAgICAgICAgICJ2YWx1ZSI6IGZhbHNlLAogICAgICAgICAgICAgICJAdHlwZSI6ICJTaG93ZXJzIgogICAgICAgICAgICB9LAogICAgICAgICAgICB7CiAgICAgICAgICAgICAgIm5hbWUiOiAiTG9ja2VycyIsCiAgICAgICAgICAgICAgInZhbHVlIjogdHJ1ZSwKICAgICAgICAgICAgICAiQHR5cGUiOiAiTG9ja2VycyIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgewogICAgICAgICAgICAgICJuYW1lIjogIlRvd2VscyIsCiAgICAgICAgICAgICAgInZhbHVlIjogZmFsc2UsCiAgICAgICAgICAgICAgIkB0eXBlIjogIlRvd2VscyIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgewogICAgICAgICAgICAgICJuYW1lIjogIkNyZWNoZSIsCiAgICAgICAgICAgICAgInZhbHVlIjogZmFsc2UsCiAgICAgICAgICAgICAgIkB0eXBlIjogIkNyZWNoZSIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgewogICAgICAgICAgICAgICJuYW1lIjogIlBhcmtpbmciLAogICAgICAgICAgICAgICJ2YWx1ZSI6IHRydWUsCiAgICAgICAgICAgICAgIkB0eXBlIjogIlBhcmtpbmciCiAgICAgICAgICAgIH0KICAgICAgICAgIF0sCiAgICAgICAgICAib3BlbmluZ0hvdXJzU3BlY2lmaWNhdGlvbiI6IFsKICAgICAgICAgICAgewogICAgICAgICAgICAgICJAdHlwZSI6ICJPcGVuaW5nSG91cnNTcGVjaWZpY2F0aW9uIiwKICAgICAgICAgICAgICAiZGF5T2ZXZWVrIjogWwogICAgICAgICAgICAgICAgImh0dHBzOi8vc2NoZW1hLm9yZy9Nb25kYXkiLAogICAgICAgICAgICAgICAgImh0dHBzOi8vc2NoZW1hLm9yZy9UdWVzZGF5IiwKICAgICAgICAgICAgICAgICJodHRwczovL3NjaGVtYS5vcmcvV2VkbmVzZGF5IiwKICAgICAgICAgICAgICAgICJodHRwczovL3NjaGVtYS5vcmcvVGh1cnNkYXkiCiAgICAgICAgICAgICAgXSwKICAgICAgICAgICAgICAib3BlbnMiOiAiMDY6MDAiLAogICAgICAgICAgICAgICJjbG9zZXMiOiAiMTM6MDAiCiAgICAgICAgICAgIH0sCiAgICAgICAgICAgIHsKICAgICAgICAgICAgICAiQHR5cGUiOiAiT3BlbmluZ0hvdXJzU3BlY2lmaWNhdGlvbiIsCiAgICAgICAgICAgICAgImRheU9mV2VlayI6IFsKICAgICAgICAgICAgICAgICJodHRwczovL3NjaGVtYS5vcmcvTW9uZGF5IiwKICAgICAgICAgICAgICAgICJodHRwczovL3NjaGVtYS5vcmcvVHVlc2RheSIsCiAgICAgICAgICAgICAgICAiaHR0cHM6Ly9zY2hlbWEub3JnL1dlZG5lc2RheSIsCiAgICAgICAgICAgICAgICAiaHR0cHM6Ly9zY2hlbWEub3JnL1RodXJzZGF5IgogICAgICAgICAgICAgIF0sCiAgICAgICAgICAgICAgIm9wZW5zIjogIjE2OjAwIiwKICAgICAgICAgICAgICAiY2xvc2VzIjogIjIwOjAwIgogICAgICAgICAgICB9LAogICAgICAgICAgICB7CiAgICAgICAgICAgICAgIkB0eXBlIjogIk9wZW5pbmdIb3Vyc1NwZWNpZmljYXRpb24iLAogICAgICAgICAgICAgICJkYXlPZldlZWsiOiBbCiAgICAgICAgICAgICAgICAiaHR0cHM6Ly9zY2hlbWEub3JnL0ZyaWRheSIKICAgICAgICAgICAgICBdLAogICAgICAgICAgICAgICJvcGVucyI6ICIwNjowMCIsCiAgICAgICAgICAgICAgImNsb3NlcyI6ICIxMjowMCIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgewogICAgICAgICAgICAgICJAdHlwZSI6ICJPcGVuaW5nSG91cnNTcGVjaWZpY2F0aW9uIiwKICAgICAgICAgICAgICAiZGF5T2ZXZWVrIjogWwogICAgICAgICAgICAgICAgImh0dHBzOi8vc2NoZW1hLm9yZy9TYXR1cmRheSIsCiAgICAgICAgICAgICAgICAiaHR0cHM6Ly9zY2hlbWEub3JnL1N1bmRheSIsCiAgICAgICAgICAgICAgICAiaHR0cHM6Ly9zY2hlbWEub3JnL1B1YmxpY0hvbGlkYXlzIgogICAgICAgICAgICAgIF0sCiAgICAgICAgICAgICAgIm9wZW5zIjogIjA4OjAwIiwKICAgICAgICAgICAgICAiY2xvc2VzIjogIjEyOjAwIgogICAgICAgICAgICB9CiAgICAgICAgICBdLAogICAgICAgICAgInNwZWNpYWxPcGVuaW5nSG91cnNTcGVjaWZpY2F0aW9uIjogWwogICAgICAgICAgICB7CiAgICAgICAgICAgICAgIkB0eXBlIjogIk9wZW5pbmdIb3Vyc1NwZWNpZmljYXRpb24iLAogICAgICAgICAgICAgICJvcGVucyI6ICIwMDowMCIsCiAgICAgICAgICAgICAgImNsb3NlcyI6ICIwMDowMCIsCiAgICAgICAgICAgICAgInZhbGlkRnJvbSI6ICIyMDIxLTAxLTAxIiwKICAgICAgICAgICAgICAidmFsaWRUaHJvdWdoIjogIjIwMjEtMDEtMzEiCiAgICAgICAgICAgIH0KICAgICAgICAgIF0KICAgICAgICB9LAogICAgICAgICJldmVudFNjaGVkdWxlIjogWwogICAgICAgICAgewogICAgICAgICAgICAiQHR5cGUiOiAiUGFydGlhbFNjaGVkdWxlIiwKICAgICAgICAgICAgInJlcGVhdEZyZXF1ZW5jeSI6ICJQMVciLAogICAgICAgICAgICAicmVwZWF0Q291bnQiOiAwLjEsCiAgICAgICAgICAgICJzdGFydERhdGUiOiAiMjAxOC0xMC0xOSIsCiAgICAgICAgICAgICJlbmREYXRlIjogIjIwMTgtMTEtMDQiLAogICAgICAgICAgICAic3RhcnRUaW1lIjogIjA4OjMwIiwKICAgICAgICAgICAgImVuZFRpbWUiOiAiMDk6MzAiLAogICAgICAgICAgICAiYnlEYXkiOiBbCiAgICAgICAgICAgICAgImh0dHBzOi8vc2NoZW1hLm9yZy9UdWVzZGF5IiwKICAgICAgICAgICAgICAiaHR0cHM6Ly9zY2hlbWEub3JnL1N1bmRheSIKICAgICAgICAgICAgXSwKICAgICAgICAgICAgImR1cmF0aW9uIjogIlBUMUgiLAogICAgICAgICAgICAic2NoZWR1bGVUaW1lem9uZSI6ICJFdXJvcGUvTG9uZG9uIgogICAgICAgICAgfQogICAgICAgIF0sCiAgICAgICAgIm9mZmVycyI6IFsKICAgICAgICAgIHsKICAgICAgICAgICAgIkB0eXBlIjogIk9mZmVyIiwKICAgICAgICAgICAgImlkZW50aWZpZXIiOiAiT1gtQUQiLAogICAgICAgICAgICAibmFtZSI6ICJPeHlnZW4gLSBBZHVsdCIsCiAgICAgICAgICAgICJwcmljZSI6IDMuMywKICAgICAgICAgICAgInByaWNlQ3VycmVuY3kiOiAiR0JQIiwKICAgICAgICAgICAgInVybCI6ICJodHRwczovL3Byb2ZpbGUuZXZlcnlvbmVhY3RpdmUuY29tL2Jvb2tpbmc_U2l0ZT0wMTQwJkFjdGl2aXRpZXM9MTQwMkNCUDIwMTUwMjE3JkN1bHR1cmU9ZW4tR0IiCiAgICAgICAgICB9LAogICAgICAgICAgewogICAgICAgICAgICAiQHR5cGUiOiAiT2ZmZXIiLAogICAgICAgICAgICAiaWRlbnRpZmllciI6ICJPWC1TTlIiLAogICAgICAgICAgICAibmFtZSI6ICJPeHlnZW4gLSBTZW5pb3IiLAogICAgICAgICAgICAicHJpY2UiOiAzLjMsCiAgICAgICAgICAgICJwcmljZUN1cnJlbmN5IjogIkdCUCIsCiAgICAgICAgICAgICJhZ2VSZXN0cmljdGlvbiI6IHsKICAgICAgICAgICAgICAiQHR5cGUiOiAiUXVhbnRpdGF0aXZlVmFsdWUiLAogICAgICAgICAgICAgICJtaW5WYWx1ZSI6IDUwCiAgICAgICAgICAgIH0sCiAgICAgICAgICAgICJ1cmwiOiAiaHR0cHM6Ly9wcm9maWxlLmV2ZXJ5b25lYWN0aXZlLmNvbS9ib29raW5nP1NpdGU9MDE0MCZBY3Rpdml0aWVzPTE0MDJDQlAyMDE1MDIxNyZDdWx0dXJlPWVuLUdCIgogICAgICAgICAgfQogICAgICAgIF0sCiAgICAgICAgImR1cmF0aW9uIjogIlBUMzBNIgogICAgICB9CiAgICB9CiAgXSwKICAibGljZW5zZSI6ICJodHRwczovL2NyZWF0aXZlY29tbW9ucy5vcmcvbGljZW5zZXMvYnkvNC4wLyIKfQ)

![Screenshot 2021-03-30 at 15 37 52](https://user-images.githubusercontent.com/2616208/113007244-f6064180-916d-11eb-8ecc-28b14f538b7d.png)

Reply:
I've removed this.